dotcom-rendering/src/components/YoutubeAtom/YoutubeAtomPlayer.tsx

```diff
@@ -490,7 +490,7 @@ export const YoutubeAtomPlayer = ({
 								player.current?.getPlayerState();
 							void playerStatePromise?.then((playerState) => {
 								if (
-									playerState &&
+									playerState != null &&
```

Comment:
This is checking for both `null` & `undefined`. Is that what we want?

Reply:
wow i've no diea where that came from! remvoing it